pkg/controller/migration/convert/network.go

```diff
@@ -5,6 +5,9 @@ import (
 	"fmt"
 	"strings"
 
+	calicocni "github.com/projectcalico/cni-plugin/pkg/types"
```

Comment:
Aren't we trying to avoid importing projectcalico/cni and projectcalico/libcalico-go?